packages/fuel-gauge/src/call-test-contract.test.ts

```diff
@@ -49,33 +48,11 @@ describe('CallTestContract', () => {
     expect(value1.toHex()).toEqual(toHex(63));
   });
 
-  it('function with empty return output configured should resolve undefined', async () => {
-    const contract = await setupContract({
-      abi: [
-        {
-          type: 'function',
-          name: 'return_void',
-          outputs: [{ type: '()', name: 'foo' }],
-        },
-      ],
-    });
-
-    const { value } = await contract.functions.return_void().call();
-    expect(value).toEqual(undefined);
-  });
-
   it('function with empty return should resolve undefined', async () => {
-    const contract = await setupContract({
-      abi: [
-        {
-          type: 'function',
-          name: 'return_void',
-        },
-      ],
-    });
+    const contract = await setupContract();
```

Comment:
All of the `abi` values passed into this function in these tests were either already existing in the default abi of the contract being set up or I added them into the sway program and rebuilt the project to have them in the abi.